catalog-info.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'openedx-events'
+  description: "This repository implements the necessary tooling and definitions used by the Hooks Extension Framework to manage the events execution and extra tools."
+  links:
+    - url: "http://github.com/openedx/openedx-events"
+      title: "Source Code"
+      icon: "Code"
+  annotations:
+    openedx.org/arch-interest-groups: "mariajgrimaldi,felipemontoya"
+spec:
+  owner: group:hooks-extension-framework
```

Comment:
@feanil in GH you can't tag `@hooks-extension-framework` you have to tag `@openedx/hooks-extension-framework`. Should that be reflected here?

Reply:
No, when we hook this up, we'll tell it to look in our org for user and group names so we don't need to have that namespacing.